docs/algorithms.rst

```diff
@@ -140,11 +145,34 @@ Compare this with Björklund's algorithm, which requires :math:`O\left((A n)^3 \
 
 .. tip::
 
-   *Implemented as* :func:`hafnian.hafnian_repeated`. The vector :math:`\mathbf{m}` is passed in the variable ``rpt``. The loop hafnian calculation can be done by passing the variable ``mu`` with the values of the vector :math:`mathbf{u}` and the option ``loops=True``.
+   *Implemented as* :func:`hafnian.hafnian_repeated`. The vector :math:`\mathbf{m}` is passed in the variable ``rpt``. The loop hafnian calculation can be done by passing the variable ``mu`` with the values of the vector :math:`\mathbf{u}` and the option ``loops=True``.
+
+Batched algorithm
+-----------------
+Using the Hermite polynomials and their connection to the matrix elements of Gaussian states and hafnians discussed in the next section one can calculate the hafnians of all the reduction of a given matrix :math:`\mathbf{A}` up to a given cutoff. The reduction of matrix :math:`\mathbf{B}` is precisely the matrix :math:`\mathbf{B}_{\mathbf{m}}` obtained by repeating (or removing) the :math:`i^{\text{th}}` row and column :math:`m_i` times. Thus given a cutoff :math:`m_{\max}` one can use the batched algorithm to calculate
+
+.. math::
+	\text{haf}\left( \mathbf{B}_\mathbf{m} \right)
+
+for all :math:`0\leq m_i < m_{\max}`, thus this function returns a tensor with :math:`n^{m_{\max}}` components.
+
+One can also use this function to calculate the same loop hafnians that Kan's algorithm returns
```

Comment:
Should make 'Kan's algorithm' a link to the Kan's algorithm section.

Reply:
It is right above it...

Reply:
Oh yeah!